### DISCUSSION TITLE
When will 4.0.7 be released?

### DISCUSSION THREAD

**Discussion by robd003** [transmission/transmission#7109] [Q&A]
@ckerr / @mikedld Do you guys have a timeline for when the bug fix will be released?

Any plan to increase cadence on the project?

**Comment by slimture** [comment]
> @ckerr / @mikedld Do you guys have a timeline for when the bug fix will be released?
> 
> 
> 
> Any plan to increase cadence on the project?

Would be nice if someone could answer this question. 4.0.6 is banned on several trackers and has been for quite a while now.

**Reply by pleasantone** [reply]
can someone on this thread point out the pr that fixed the bug that's getting the client banned?

**Reply by andreygursky** [reply]
PR: https://github.com/transmission/transmission/pull/7086

**Comment by Nemric** [comment]
Hi, please you should publish some mea culpa like : https://github.com/utkuozdemir/pv-migrate?tab=readme-ov-file#pv-migrate that can be understood ... but perhaps should I go to @Coeur repo and build his/her "with fix" branch ... and that will become the new transmission official repo for me and others that use my OCI image !?

**Reply by Coeur** [reply]
I can't maintain a fork alone.
You may use my fix branch for a 4.0.7 workaround for now, but you'd still better stay on https://github.com/transmission/transmission for future development as long as ckerr is still merging PRs from time to time.

**Comment by Nemric** [comment]
Well, I'm now running the 4.1.0-dev that I build in container images every weeks on sunday with the main tag : https://hub.docker.com/r/nemric/transmission/tags

**Reply by holta** [reply]
We need a weekly .deb file for Ubuntu / Debian / Linux Mint / Raspberry Pi OS / Pop! OS just exactly like that (-:

So more people can hopefully assist with TDD :pray:

**Comment by ilf** [comment]
Debian [announced](https://lists.debian.org/debian-devel-announce/2025/01/msg00004.html) the [freeze dates](https://release.debian.org/#release-dates) for the upcoming Debian 13: [soft freeze](https://release.debian.org/testing/freeze_policy.html#soft) is **2025-04-15** - in less than three months.

It would be great to have transmission 4.0.7 in Debian 13. For that, it would help to have a 4.0.7 release in the next month or so, so the [maintainers](https://tracker.debian.org/pkg/transmission) have enough time to package and test it.

Thanks a lot, everyone!

**Reply by Coeur** [reply]
Considering the lack of communication from the admins, I'm pretty sure there will never be a 4.0.7 release.

**Reply by ilf** [reply]
The Debian soft freeze is now six weeks away. A release needs to be packaged, uploaded and transition to testing. This takes at least ten days, more likely two weeks.

@ckerr: Could you release 4.0.7 in March? That would be great! Thanks a lot <3

**Reply by ilf** [reply]
The Debian 13 [freeze has officially started](https://lists.debian.org/debian-devel-announce/2025/03/msg00011.html).

Currently, Debian unstable and testing include [transmission 4.0.6](https://packages.debian.org/source/unstable/transmission). If nothing happens, Debian 13 stable will include this version if transmission. Unfortunately, transmission 4.0.6 is currently blocked on several trackers.

Transmission released [4.1.0-beta.2](https://github.com/transmission/transmission/releases/tag/4.1.0-beta.2) two weeks ago, and that's currently in [Debian experimental](https://tracker.debian.org/pkg/transmission). But that will *not* migrate to testing and become stable.

@ckerr: Would it be possible to please release 4.0.7 (or 4.1.0) soon? I fear that if that doesn't happen within the next few days, Debian 13 will have a version of transmission that several trackers block.

Thanks a lot <3

**Reply by ile6695** [reply]
Good news. 4.1.0-beta.2 is now in Debian testing repository [https://tracker.debian.org/pkg/transmission](https://tracker.debian.org/pkg/transmission)
It migrated today from unstable/sid.

**Reply by ilf** [reply]
Debian 13 was released three days ago: https://www.debian.org/releases/trixie/
It includes transmission 4.1.0~beta2+dfsg-3: https://packages.debian.org/trixie/transmission

Unfortunately, many trackers ban beta versions of torrent clients. :(

Could we please get a 4.1.0 release, so we can get that into 13.1 due in a month?

**Comment by robd003** [comment]
It's officially been 11 months. What is going on with this project?!?

**Comment by pipatron** [comment]
A user is now trying to fork this project so these issues can be solved: https://github.com/transmission/transmission/discussions/7707

**Reply by ilf** [reply]
> A user is now trying to fork this project so these issues can be solved: #7707

No, a potential fork does *not* solve this issue. I have serious doubts a new fork will succeed. For anyone who wants to migrate, go ahead. But people will stay with the original transmission. For them, this issue remains open.

**Reply by user-1-2-3-4-5** [reply]
> A user is now trying to fork this project so these issues can be solved: #7707

Do you expect that the fork will be accepted on private trackers, unless it eventually supersedes the original release?

**Comment by ilikepeaches** [comment]
relevant: https://github.com/transmission/transmission/issues/7623
